src/include/duckdb/common/enums/active_transaction_state.hpp

```diff
@@ -0,0 +1,15 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/common/enums/active_transaction_state.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+namespace duckdb {
+
+enum class ActiveTransactionState { UNSET, OTHER_ACTIVE_TRANSACTIONS, NO_OTHER_TRANSACTIONS };
```

Comment:
nit: maybe `UNSET, OTHER_TRANSACTIONS, NO_OTHER_TRANSACTIONS`. We already have the `active` part covered by the `enum` name